test/blackbox-tests/test-cases/oxcaml/dune

```diff
@@ -0,0 +1,3 @@
+(cram
+ (applies_to :whole_subtree)
+ (enabled_if %{oxcaml:supported}))
```

Comment:
Why do you need a macro? can't it just be a variable?

Reply:
Indeed, it can.